scico/linop/xray/_xray.py

```diff
@@ -193,3 +195,128 @@ def _calc_weights(x0, dx, nx, angle, y0):
     weights = jnp.minimum(distance_to_next, width) / width
 
     return inds, weights
+
+
+class Parallel3dProjector:
+    """General-purpose, 3D, parallel ray X-ray projector."""
+
+    def __init__(
+        self,
+        input_shape: Shape,
+        matrices: ArrayLike,
```

Comment:
Can we add a parameter equivalent to `det_spacing` for `xray.astra.XRayTransform3D.__init__`?